src/main/java/com/it4045/pokemonviewer/enterprise/dto/Pokemon.java

```diff
@@ -3,51 +3,52 @@
 import com.google.gson.annotations.SerializedName;
 import lombok.Data;
 
+import java.lang.reflect.Type;
 import java.util.List;
 
 public @Data
 class Pokemon {
 
     @SerializedName("id")
-    private int pokemonId;
+    private int id;
     @SerializedName("num")
-    private String pokemonNumber;
+    private int number;
```

Comment:
number should be integer instead of String.

Reply:
Yes.  Renames look good as well.